memiavl/db.go

```diff
@@ -898,8 +902,8 @@ func walPath(root string) string {
 //
 // current -> snapshot-0
 // ```
-func initEmptyDB(dir string, initialVersion uint32) error {
-	tmp := NewEmptyMultiTree(initialVersion, 0)
+func initEmptyDB(dir string, initialVersion uint32, snapshotWriterLimit int) error {
```

Comment:
we can use a constant in this method to keep it simpler.

Reply:
> we can use a constant `1` in this method to keep it simpler.

fix